infra/cifuzz/cifuzz_test.py

```diff
@@ -361,5 +361,54 @@ def test_get_invalid_project(self):
     self.assertIsNone(cifuzz.get_latest_cov_report_info(''))
 
 
+class KeepAffectedFuzzersUnitTest(unittest.TestCase):
```

Comment:
also add a test for a case where we don't have coverage for a fuzzer. In real like it could be a new fuzz target which was added or renamed just today

Reply:
Added.